resources/playbook.yml

```diff
@@ -89,11 +89,17 @@
         - initial_provision
 
     - name: Install bats
-      shell: |
-        git clone https://github.com/bats-core/bats-core.git
-        cd bats-core
-        sh ./install.sh /usr/local
-      ignore_errors: true
+      block:
```

Comment:
This may actually be a problem because I remember that CL `bats` package has code changes in it